### ISSUE TITLE
Where is the players visible range setting on the map?

### ISSUE THREAD

**Issue by itsportspe** [endless-sky/endless-sky-wiki#72]
I'm in the process or making a plugin, and I couldn't find where the setting for the the players visible range diameter on the map is. See attached image.
![endless_sky_visible_range](https://github.com/user-attachments/assets/bd57fac8-f9ed-471c-82c9-35acc4df90b4)

**Comment by TheGiraffe3** [comment]
It's the `jump range` attribute?

**Comment by tibetiroka** [comment]
There are two ranges. One is the jump range, the other is the view range. By default, they are the same, so the two circles overlap. If you are in a system with a different jump range however (or have a drive with a different range), you will see two distinct circles, one of them being the view range.

**Comment by itsportspe** [comment]
> It's the `jump range` attribute?

This is incorrect. There are 2 attributes jump range is one of them, and I assume visible range is the other one, but I don't know what it's actually called because I couldn't find it. In the the attached image jump range is labeled in white, and visible range is labeled in yellow.
![endless_sky_visible_range2](https://github.com/user-attachments/assets/2a4b2717-3f6c-4b39-8b7c-4e1ace61c914)

**Comment by tibetiroka** [comment]
>  The problem is that you can increase the jump range, but you wont ever be able to jump further than the visible range.

I think this was fixed in #10704.

**Comment by itsportspe** [comment]
> > The problem is that you can increase the jump range, but you wont ever be able to jump further than the visible range.
> 
> I think this was fixed in [endless-sky/endless-sky#10704](https://github.com/endless-sky/endless-sky/pull/10704).

For jump range this seems to be fixed, but I'm trying to increase visible range. Any undiscovered system that comes within visible range will show on the map.

**Comment by tibetiroka** [comment]
I don't believe that is supported at the moment.

**Comment by TomGoodIdea** [comment]
As tibetiroka said, this is a request for a new feature in the game, so it should be on the main repo rather than here. Also there already is https://github.com/endless-sky/endless-sky/issues/5551, so you don't need to open another issue.
